DMTN-036.tex

```diff
@@ -0,0 +1,808 @@
+# we are ready to run. Initialize parameters by running "partial fits"
+        fit.Minimize("Distortions")
+        fit.Minimize("Positions")
+# now fit both sets simultaneously.
+        fit.Minimize("Distortions Positions")
+# output residual tuples
+        fit.MakeResTuple("res.list")
+\end{verbatim}
+
+\appendix
+
+\section{Representation of distortions in SIP WCS's \label{sec:sip-wcs}}
+
+The purpose of the appendix is to provide the minimal introduction
+to WCS concepts required to understand the code (and the comments)
+when browsing through it. Readers familiar with WCSs can give up here.
+
+WCS's are abstract concepts meant to map data on coordinate systems.
+In the astronomical imaging framework, this almost always means mapping
+the pixel space into sidereal coordinates, expressed in some conventional
+space\footnote{The WCS concepts are broad enough to accommodate mapping of planet
+images, but we will obviously not venture into that.}. One key aspect of the WCS ``system'' is that it proposes some
+implementation of the mappings in FITS headers, which comes with software
+libraries to decode and encode the mappings. The WCS conventions cover
+a very broad scope of applications, and wide-field imaging makes use
+of a very small subset of those.
+
+For the mappings used in wide-field imaging, the transformation from
+pixel space to sky can be pictured in two steps: 
+\begin{enumerate}
+\item mapping coordinates in pixel space onto a plane. 
+\item de-projecting this plane to the celestial sphere. 
+\end{enumerate}
+Let us clear up the projection/de-projection step first. There are
+plenty of choices possible here, and the differences only matter fir
+really large images. The projection used by default in the imaging
+community seems to be the gnomonic projection: the intermediate space
+is a plane tangent to the celestial sphere and the plane$\rightarrow$sphere
+correspondence is obtained by drawing lines that go through the center
+of the sphere. In practice there is no need to know that, because
+any software dealing with WCS's can pick up the right FITS keywords
+and compute the required projection and de-projection. For this gnomonic
+projection, one finds \verb&CTYPE1='RA---TAN'& and \verb&CTYPE2='DEC--TAN'&
+in the FITS header. This projection is often used to generate re-sampled
+and/or co-added images and one should keep in mind that, for large
+images, the pixels are not exactly iso-area. One point of convention
+that might be useful to keep in mind; is that WCS conventions express
+angles in degrees. In the gnomonic projection, offsets in the tangent
+plane are expressed in degrees (defined through angles along great
+circles at the tangent point), so that the metric in the tangent plane
+is ortho-normal), and sidereal angles evaluated on the sky are also
+provided in degrees by the standard implementations. A notable exception
+is the LSST software stack where, by default, the angles are provided
```

Comment:
AST also uses radians.

Reply:
I'm not sure about the utility of this whole section, so I'm going to leave it for now.